lib/components/mobile/batch-search-screen.tsx

```diff
@@ -14,6 +23,20 @@ import MobileNavigationBar from './navigation-bar'
 
 const { SET_FROM_LOCATION, SET_TO_LOCATION } = MobileScreens
 
+const MobileSearchSettings = styled.div<{ advancedPanelOpen: boolean }>`
+  background: white;
```

Comment:
<img width="480" alt="Screenshot 2024-08-13 at 11 27 45 AM" src="https://github.com/user-attachments/assets/29904b6b-f32c-4942-ab67-1ac51f164ac9">
Seeing a weird gray strip when setting the sidebar background color to a color other than white. My guess is this line is the culprit

Reply:
Do you know specifically what panel this is? I've tried resetting the background on a few different sidebar layers and im not seeing this!